docs/guides/quickstarts/react.mdx

```diff
@@ -1,149 +1,308 @@
 ---
-title: Setup Nhost with React
+title: Get up and running with Nhost and React
 sidebarTitle: React
-description: Get up and running with Nhost and React
```

Comment:
I think I like better the old description

Reply:
👌 will revert it back to `Get up and running with Nhost and React`